ohybridproxy/Makefile

```diff
@@ -8,16 +8,18 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ohybridproxy
 PKG_SOURCE_VERSION:=0dfef1eb5f067250a5f24a899536879ea4fdc4c5
-PKG_VERSION:=2020-05-22-$(PKG_SOURCE_VERSION)
-PKG_RELEASE:=1
+PKG_SOURCE_DATE:=2020-05-22
```

Comment:
Don't we typically group the SOURCE_DATE and SOURCE_VERSION together with the other PKG_SOURCE_* variables?

Reply:
Oops, this is openwrt-routing, sorry if I invaded somebody else's realm or judgement here ...

Reply:
I do this to keep each section small.

It makes sense to put date and version at the top anyway.